stories/core/LabeledInput.stories.tsx

```diff
@@ -146,3 +147,29 @@ Inline.args = {
   status: 'negative',
   displayStyle: 'inline',
 };
+
+export const HybridLayout: Story<LabeledInputProps> = ({
```

Comment:
I think we also need to show input + button.

Reply:
you think create another story?
Maybe put another input in the same..?

Reply:
I would say separate story. We don't really have several cases in one story.

Reply:
yeah, I just started to think to introduce some :D Might be easier to have a look, but..

Reply:
😁
![image](https://user-images.githubusercontent.com/36186912/133742915-9ba6e4e0-0118-4617-a907-934193a75cdc.png)

Reply:
added